app/timetables/models.py

```diff
@@ -175,3 +175,16 @@ def __str__(self):
 
     class Meta:
         unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+
+class Vendor(SlugifyMixin, TimestampMixin):
+    """Model representing food service-provider"""
+
+    name = models.CharField(max_length=255)
+    slug = models.SlugField(max_length=255, unique=True, editable=False)
```

Comment:
@andela-eisaac: I will create an issue for it. Please, let's complete this review.